runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/io/UnboundedSourceWrapper.java

```diff
@@ -158,7 +159,7 @@ public void run(SourceContext<WindowedValue<OutputT>> ctx) throws Exception {
     }
 
     LOG.info("Unbounded Flink Source {}/{} is reading from sources: {}",
-        subtaskIndex,
+        subtaskIndex + 1,
```

Comment:
Yes, it is confusing. Either we start the index at 1 or we change the message to something like "Unbounded Flink Source with id {} ({} in total)".